warehouse/templates/pages/sponsor.html

```diff
@@ -0,0 +1,244 @@
+ # limitations under the License.
+-#}
+{% extends "base.html" %}
+{% block title %}{% trans %}Sponsor the Python Packaging Working Group{% endtrans %}{% endblock %}
+
+{% block content %}
+<div class="horizontal-section">
+  <div class="site-container">
+    <div class="sponsorship-intro">
+      <div class="sponsorship-intro__text">
+        <div class="heading-wsubtitle">
+          <h1 class="heading-wsubtitle__heading">{% trans %}Sponsor PyPI and related projects{% endtrans %}</h1>
+          <p class="heading-wsubtitle__subtitle">{% trans %}Donate to the Python Packaging Working Group{% endtrans %}</p>
+        </div>
+        <p class="lede-paragraph">
+          {% trans href='https://wiki.python.org/psf/PackagingWG', title=gettext('External link') %}The <a href="{{ href }}" title="{{ title }}" target="_blank" rel="noopener">Python Packaging Working Group</a> is a work group of the Python Software Foundation which raises and distributes funds to improve Python's packaging ecosystem.{% endtrans %}
+        </p>
+        <p>{% trans %}Recent projects funded through the working group include:{% endtrans %}</p>
+        <ul class="hooray-list">
+          <li>{% trans %}The successful relaunch of the Python Package Index, powered by the new 'Warehouse' codebase{% endtrans %}</li>
+          <li>{% trans %}Adding security features to PyPI, including two-factor authentication{% endtrans %}</li>
+          <li>{% trans %}Improving PyPI's web interface for users with disabilites, and adding support for multiple locales{% endtrans %}</li>
+          <li>{% trans %}Improving pip's dependency resolver and user experience{% endtrans %}</li>
+        </ul>
+        <p>
+          {% trans %}With your support, the working group can continue to fund packaging improvements, benefiting millions of Python users around the world.{% endtrans %}
+        </p>
+      </div>
+      <div class="sponsorship-intro__donations sponsor-package">
+        <div class="sponsor-package__header">
+          <div class="sponsor-package__icon">
+            <i class="fas fa-donate" aria-hidden="true"></i>
+          </div>
+          <h2>{% trans %}Community donations{% endtrans %}</h2>
+        </div>
+        <div class="sponsor-package__body">
+          <p>{% trans %}We greatly appreciate both one-time and recurring donations from individuals and small businesses.{% endtrans %}</p>
+          <p>{% trans href='https://www.python.org/psf/donations/', title=gettext('External link') %}For US taxpayers, your <a href="{{ href }}" title="{{ title }}" target="_blank" rel="noopener">donation may be tax-deductible</a>.{% endtrans %}</p>
+          <p>{% trans %}Every donation counts!{% endtrans %}</p>
+        </div>
+        <div class="sponsor-package__button">
+          <a class="button" href="https://donate.pypi.org">{% trans %}Donate here{% endtrans %}</a>
+          <p>{% trans href='#sponsorship-packages' %}Or <a href="{{ href }}">become a sponsor</a>{% endtrans %}</p>
```

Comment:
@dstufft the "individuals and small businesses" is trying to differentiate between donations below 5k and those over, which would be classified as a "sponsor".  What do you think about removing that text, and updating the text under the button?  e.g.

```suggestion
          <p>{% trans href='#sponsorship-packages' %}Donating over $5,000? <a href="{{ href }}">Become a sponsor</a>{% endtrans %}</p>
```